package.json

```diff
@@ -10,6 +10,7 @@
   },
   "dependencies": {
     "@types/graphql": "^14.0.3",
+    "googleapis": "^36.0.0",
```

Comment:
this would need to be added to peril's dependencies - which I think is reasonable